pandas/io/gbq.py

```diff
@@ -125,13 +123,15 @@ def read_gbq(
         ``fastavro`` packages.
 
         .. versionadded:: 0.25.0
-    progress_bar_type : Optional, str
+    progress_bar_type : str
         If set, use the `tqdm <https://tqdm.github.io/>`__ library to
         display a progress bar while the data downloads. Install the
         ``tqdm`` package to use this feature.
 
         Possible values of ``progress_bar_type`` include:
 
+        ``NOT_SET``
+            Arg is not set by caller, don't including the argument in pandas-gbq call.
```

Comment:
Does the None case still need to be documented?

Reply:
Kept the documentation for `None` because on passing None as a value the caller can achieve the functionality of having no progress bar, which is what the original issue is as per the description.

Reply:
we cannot change this, pls revert.